PatternPal/PatternPal.Core/PatternPal.Core.csproj

```diff
@@ -54,4 +54,8 @@
     <Protobuf Include="$(RepositoryRoot)Protos/logprovider.proto" GrpcServices="Server" Link="Protos/logprovider.proto" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="CodeStates\" />
```

Comment:
Why are you including this folder? Is this the folder where the code states send by the users of the extension end up? If so, that shouldn't be stored in the source code of the program, because that directory won't exist when you install the program.